samples/dnn/yolo_object_cam.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+Sample of using OpenCV dnn module in real time with device capture (based in yolo_object_detection.cpp)
+Author: Alessandro de Oliveira Faria cabelo@opensuse.org
+*/
+
+
+#include <opencv2/dnn.hpp>
+#include <opencv2/dnn/shape_utils.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+#include <iostream>
+
+#include "opencv2/opencv_modules.hpp"
+
+using namespace cv;
+using namespace cv::dnn;
+
+#include <fstream>
+#include <iostream>
+#include <algorithm>
+#include <cstdlib>
+#include <list>
+
+using namespace std;
+
+// COMPILE:
+// g++ `pkg-config --cflags opencv` `pkg-config --libs opencv` yolo_object_detection.cpp -o yolo_object_detection
+
+// RUN:
+// yolo_object_cam -cam=0  -cfg=[PATH-TO-DARKNET]/cfg/yolo.cfg -model=[PATH-TO-DARKNET]/yolo.weights   -labels=[PATH-TO-DARKNET]/data/coco.names
+const size_t network_width = 416;
+const size_t network_height = 416;
+
+const char* about = "This sample uses You only look once (YOLO)-Detector "
+                    "(https://arxiv.org/abs/1612.08242)"
+                    "to detect objects on capture device\n"; 
+
+const char* params
+    = "{ help           | false | print usage         }"
+      "{ cam            |       | device index        }"
+      "{ cfg            |       | model configuration }"
+      "{ model          |       | model weights       }"
+      "{ labels         |       | label of the object }"
+      "{ min_confidence | 0.24  | min confidence      }";
+
+
+void assetRoi(Rect &_roi, Mat &_frame  )
```

Comment:
Add "static" for local functions "assetRoi" / "addLabels" / "returnLabel"

Reply:
Done